src/exchanges/cache.ts

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable @typescript-eslint/no-use-before-define */
```

Comment:
I actually did disable the rules in the exchanges since it would've produced a very large diff. So I guess we should have a discussion about whether we disable this rule globally or follow it globally.

Reply:
I think in some cases we should just reorder it in a separate commit, so we have an easily trackable change history of what happened in these files

Reply:
🆒I'll revise so that this commit just disables the reordering and we can reorder in a separate commit.